projects/ngx-d3timeline/src/lib/ngx-d3timeline.service.ts

```diff
@@ -79,4 +92,18 @@ export class NgxD3TimelineService implements OnDestroy {
   private zoomed() {
     this.store.dispatch(new fromActions.ZoomedAction(event));
   }
+
+  setupResizing(hostElement: ElementRef, changeDetector: ChangeDetectorRef) {
+    const resizes$ = createResizeObservable(hostElement.nativeElement);
+
+    resizes$
+      .pipe(takeUntil(this.destroySubject), debounceTime(100))
+      .subscribe(view => {
+        this.setView(view);
```

Comment:
maybe move this to an updateView method?

```typescript
  setupResizing(hostElement: ElementRef, changeDetector: ChangeDetectorRef) {
    const resizes$ = createResizeObservable(hostElement.nativeElement);

    resizes$
      .pipe(takeUntil(this.destroySubject), debounceTime(100))
      .subscribe(this.updateView.bind(this));
  }

  private updateView(view: [number, number]) {
    this.setView(view);

    // possibly necessary due to current lack of monkey patching for ResizeObserver
    // https://dev.to/christiankohler/how-to-use-resizeobserver-with-angular-9l5
    this.changeDetector.detectChanges();
  }
```

Reply:
done